### PR TITLE
Release 0.13.0

### PR DESCRIPTION
**Pull request by pengyin-shan**
I put tomorrow (2023-12-14) as the deployment date in case we cannot finish this today. This release will include:

### Added

- Added Google Analytics using `react-ga` module to page router [#217](https://github.com/policy-design-lab/pdl-frontend/issues/217)

### Changed

- Updated all maps to show exact number of dollars instead of rounding units to the nearest million [#137](https://github.com/policy-design-lab/pdl-frontend/issues/137)

- Replaced the arrow of table to Material UI icons to show in Linux browser[#151](https://github.com/policy-design-lab/pdl-frontend/issues/151)

### CODE REVIEW

**Comment by sandeep-ps** [comment]
There seems to be a minor alignment issue related to the arrows. Please see below.

Production instance: 
image

Development instance:
image

This is using Chrome browser on MacOS. Otherwise, the changes look fine to me. Thanks.

**Comment by navarroc** [comment]
I see the same alignment except I don't get the issue with the last one (payment pct nationwide). Here's how it looks on mine. 
Screenshot from 2023-12-14 11-44-50

**Comment by pengyin-shan** [comment]
@navarroc @sandeep-ps  : this is how the icon looks in default in Material UI. Do you want me to use another library?

**Comment by pengyin-shan** [comment]
Material UI is the default React library that we normally go to, but I can use other libraries also

**Comment by navarroc** [comment]
> @navarroc @sandeep-ps : this is how the icon looks in default in Material UI. Do you want me to use another library?

I think it looks ok to me.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
...
+## [0.13.0] - 2023-12-14
+
+### Added
+
+- Added Google Analytics using `react-ga` module to page router [#217](https://github.com/policy-design-lab/pdl-frontend/issues/217)
+
+### Changed
+
+- Updated all maps to show exact number of dollars instead of rounding units to the nearest million [#137](https://github.com/policy-design-lab/pdl-frontend/issues/137)
+
```
Comment (navarroc) [inline CHANGELOG.md:16]: Can you remove the empty line?

Comment (pengyin-shan) [reply CHANGELOG.md:16]: Sorry you mean the empty line between lines? Or line 16?

**Comment by navarroc** [comment]
Line 16, it looks like in some cases we've put an extra line between changelog entries and sometimes we don't. It's not a big deal, just cosmetic.

**Comment by pengyin-shan** [comment]
> Line 16, it looks like in some cases we've put an extra line between changelog entries and sometimes we don't. It's not a big deal, just cosmetic.

Ah I see! Yes, I removed space between two list markdowns

**Review by navarroc — APPROVED** [review]

**Review by sandeep-ps — APPROVED** [review]

**Pull request merged**